# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/coreutils/dateseconds.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./lib/getdate.y	10 Sep 2003 08:37:15 -0000	1.80
+++ ./lib/getdate.y	15 Mar 2004 08:30:57 -0000
@@ -220,6 +220,7 @@ time:
       {
 	PC.hour = $1.value;
 	PC.minutes = $3.value;
+	PC.seconds = 0;
 	PC.meridian = MER24;
 	PC.zones_seen++;
 	PC.time_zone = $4.value % 100 + ($4.value / 100) * 60;
